### ISSUE TITLE
embed list of lints into the README file on every push to master

### ISSUE THREAD

**Issue by yegor256** [objectionary/lints#65]
Let's create a GHA job that will, on every time we push to master, instantiate all lints and collect their "motive()"-s (`.md` files). Then, it will put all these Markdown files into the `gh-pages` branch, in the `motives` directory, and push. Then, it will modify the README file, inserting a list of all lints into it. Currently, we have a list there, but it's static. This one should be removed and a dynamically generated one should take its place.

**Comment by Marat-Tim** [comment]
Let me try to do it

**Comment by Marat-Tim** [comment]
@yegor256 Is the task relevant? Wouldn't it be better to add a test that verifies that for each lint from the `PkMono` and `PkWpa` classes there is a corresponding motive and that for each motive there is a lint?

**Comment by Marat-Tim** [comment]
@yegor256 while working on [this](https://github.com/objectionary/lints/pull/505), I got the idea that we need to change the approach to providing a motive file and provide more than just a file.md(because I have to add to the documentation exactly which rules from markdownlint we ignore, what is stored in my variable). We need a template (for example, freemarker) to which variables can be passed. And then add a job that collects all the templates in a regular ci-cd pipeline, substitutes variables in them and checks that the resulting files are valid from the point of view of markdown. And add a job that, when pushed to master, dynamically collects files in the gh-pages branch

If we want to do this, then getting all the lines in the runtime makes sense

**Comment by yegor256** [comment]
@Marat-Tim it's better to propose new ideas in separate tickets

**Comment by yegor256** [comment]
@Marat-Tim yes, this task is relevant. We want to see a full list of lints in the README file. It's not about testing, it's about showing our users what we have in the library.

**Comment by Marat-Tim** [comment]
@yegor256 We are already showing a list of what we have in the gh-pages branch. Do you mean that we need to add this list to README.md in the master branch?

**Comment by yegor256** [comment]
@Marat-Tim yes, we want to see this list in the README too

**Comment by Marat-Tim** [comment]
@yegor256 I'll clarify the requirements. We want the list of lints to be added to README.md in the `master` branch? Or we want to add README.md to the `gh-pages` branch. And which files should be referenced in this README? To html pages from the `gh-pages` branch or to some other ones?

**Comment by yegor256** [comment]
@yegor256 we want to update the README in the master branch. We want to list _all_ lints that we have in the repo: they can be found by simply iterating `.md` motive files.

**Comment by yegor256** [comment]
@Marat-Tim I agree, let's add a link

**Comment by VasilevNStas** [comment]
@yegor256  @volodya-lombrozo @maxonfjvipon

 I think this one can be closed. Two reasons:
1. There is no static list of lints in the README anymore — the current "Design of This Library" section only describes the architecture, so there is nothing left to replace with a dynamically generated list.
2. The list of lints with their motives is already generated automatically on every tag by the `motives.yml` workflow (see `.github/motives.sh`), which publishes the full list to GitHub Pages.
If a list in the README itself is still desired, a small workflow reusing `PkMono`/`PkByXsl` from the classpath could produce a markdown table between README markers, but the value seems low given the gh-pages list already exists.
